azext_iot/tests/iothub/certificate/test_iothub_certificate_root_authority_int.py

```diff
@@ -9,23 +9,41 @@
 
 class TestIotHubCertificateRoot(IoTLiveScenarioTest):
     def __init__(self, test_case):
+        self.original_enableRootCertificateV2 = False
         super(TestIotHubCertificateRoot, self).__init__(test_case, add_data_contributor=False)
 
-    def test_certificate_root(self):
+    def tearDown(self):
+        # Revert transition
+        version = 'v2' if self.original_enableRootCertificateV2 else 'v1'
         self.cmd(
-            f"iot hub certificate root-authority show -n {self.entity_name} -g {self.entity_rg}",
-            checks=[
-                self.check("enableRootCertificateV2", False),
-            ],
+            f"iot hub certificate root-authority set -n {self.entity_name} -g {self.entity_rg} --cav {version} --yes"
         )
 
-        # Transition to Digicert (initial transition)
-        self.cmd(
-            f"iot hub certificate root-authority set -n {self.entity_name} -g {self.entity_rg} --cav v2 --yes",
-            checks=[
-                self.check("enableRootCertificateV2", True),
-            ],
-        )
+        super(TestIotHubCertificateRoot, self).tearDown()
+
+    def test_certificate_root(self):
+        authority = self.cmd(
+            f"iot hub certificate root-authority show -n {self.entity_name} -g {self.entity_rg}"
+        ).get_output_in_json()
+
+        self.original_enableRootCertificateV2 = authority["enableRootCertificateV2"]
+
+        if authority["enableRootCertificateV2"] is False:
```

Comment:
nit: you can do 
if not authority["enableRootCertificateV2"]:

Reply:
got it!